<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>West1</class>
 <widget class="QWidget" name="West1">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1285</width>
    <height>736</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
     background-color:rgb(128, 128, 128);
}</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="titleLabel">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Players to be excluded</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>-1</number>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTableView" name="tableView_2"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>                  </string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QTableView" name="tableView"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="img1">
         <property name="text">
          <string>gxjshxkccjhbjhnsjlchjdslshshsdggddcchbjbjjjnbh</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Team Roster</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Potential newcomers</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>                  </string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QTableView" name="tableView_4"/>
         </item>
         <item row="0" column="0">
          <widget class="QTableView" name="tableView_3"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="exit1Button">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Main Page</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
